src/Shouldly.Tests/Shouldly.Tests.csproj

```diff
@@ -17,9 +15,8 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
-    <PackageReference Include="PublicApiGenerator" Version="8.0.1" />
+    <PackageReference Include="PublicApiGenerator" Version="10.0.0-beta1" />
     <PackageReference Include="TestStack.ConventionTests" Version="3.0.1" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
```

Comment:
`dotnet-test` was discontinued: `https://github.com/xunit/xunit/issues/1835#issuecomment-431199286`

Version 2.3.1 has no rollforward policy and can only run on .NET Core 2.0 which is EOL.